feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64ec96761156a187ed32b274.md

---
id: 64ec96761156a187ed32b274
title: Step 22
challengeType: 0
dashedName: step-22
---

# --description--

> You need to have a delete and edit button to delete and edit each of the tasks.

Reviewer: ```suggestion
To allow for task management, you need to include both a delete and an edit button for each task.
```

> Create two `button` elements with the type `button` and class `btn`. Set the text of the first button to `Edit` and the second to `Delete`.

Reviewer: ```suggestion
Create two `button` elements with the `type` attribute set to `button` and the `class` attribute set to `btn`. Set the text of the first button to `Edit` and the text of the second button to `Delete`.
```
